src/utils/common.js

```diff
@@ -27,10 +25,10 @@ export const isNull = (value) => {
 
 export const isDefinedAndNotNull = (value) => {
   const values = createArrayFromValue(value);
-  return values.every(item => isDefined(item) && item !== null);
+  return values.every(item => isDefined(item) && !isNull(item));
 };
 
 export const isDefinedAndNull = (value) => {
   const values = createArrayFromValue(value);
```

Comment:
Nit: you don't really need `createArrayFromValue()` here, because `isDefined` and `isNull` both call it.

Reply:
I think I'm going to leave `createArrayFromValue` in here, for a couple reasons:
1. Consistency between these types of util functions
2. Do the conversion of `value` --> array once upfront instead of 2 times for both `isDefined` and `isNull`.